automated-comments/go/raindrops/all_combinations_implemented.md

```diff
@@ -0,0 +1,4 @@
+Although this solution passes the tests, it's too tightly coupled to the specific cases.
+Imagine all the new combinations that would need to be added if one or more extra cases are requested.
+It would rapidly become difficult to maintain.
+Try concatenating the strings e.g. using the `+=` operator.
```

Comment:
I suggest 'for example' instead of 'e.g.'; this is much clearer to non-native English speakers.